code/game/objects/items.dm

```diff
@@ -632,3 +654,46 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		M.dirty++
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
+
+/obj/item/proc/set_force_string()
+	var/roundForce = round(force)

```

Comment:
save a proc call by removing this and increasing the second number for all of these but the last(`20 to 25`) by 1